cmd/influxd/inspect/dump_tsm/dump_tsm_test.go

```diff
@@ -96,7 +96,6 @@ func Test_DumpTSM_ManyKeys(t *testing.T) {
 		expectOuts: makeExpectOut(
 			[]string{"Total: 3", "Size: 102"},
 			[]string{"Blocks:", "float64", "s8b/gor", "9/19"},
-			[]string{"cpu", "foobar", "mem", "float64"},
```

Comment:
Before we go through with this, do we know why these checks were flaky? Do we expect anything in the tool to be nondeterministic?

Reply:
I'm not really sure, I want to look into this more but also don't want to leave master branch failing tests occasionally so I'd like to investigate locally for a while

Reply:
In that case, instead of outright deleting these lines can you:
* Make a ticket to investigate the failures
* Comment out these lines
* In a comment above the line, write a `TODO` explaining the problem & including a link to the GitHub issue

That way there's a record of what we saw, in case you get pulled away before you have time to fully investigate